src/Microsoft.Bcl.HashCode/src/Microsoft.Bcl.HashCode.csproj

```
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetMinimum);netstandard2.1;netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <IsPackable>false</IsPackable>
+    <Nullable>enable</Nullable>
```

Let's keep the packaging properties together:

```suggestion
    <Nullable>enable</Nullable>
    <IsPackable>false</IsPackable>
```